app/aws-lsp-json-runtimes/src/tests/jsonServerCFInteg.test.ts

```diff
@@ -141,4 +151,55 @@ describe('Test JsonServer with CloudFormation schema', () => {
 
         expect(result).to.deep.equal(FORMAT_EDITS_JSON)
     })
-})
+}),
+    describe('Test JsonServer with CloudFormation schema and a custom implementation of the JsonLanguageService', () => {
+        let client: LspClient
+        let endpoint: JSONRPCEndpoint
+        let process: ChildProcessWithoutNullStreams
+        const runtimeFile = path.join(__dirname, '../../', 'build', 'aws-lsp-json-standalone-with-customization.js')
+
+        before(async () => {
+            ;({ client, endpoint, process } = await createLSPServer(runtimeFile))
```

Comment:
Is the semicolon (`;`) needed?

```suggestion
            ({ client, endpoint, process } = await createLSPServer(runtimeFile))
```

Reply:
Same as before - auto added from a vscode formatter, will update.